packages/api/src/models/Market.ts

```diff
@@ -35,6 +35,12 @@ export class Market {
   @Column({ type: 'varchar' })
   address: string;
 
+  @Column({ type: 'varchar' })
+  vaultAddress: string;
+
+  @Column({ type: 'boolean', default: false })
+  isYin: boolean;
```

Comment:
np: This fixes the architecture for only 2 markets. An id to identify the order would be better (vaults support N markets)

Reply:
Good point, I suggest we keep as-is here for now for simplicity, can easily migrate when needed